tools/wpt/browser.py

```diff
@@ -445,10 +445,10 @@ def install_geckodriver_nightly(self, dest):
     def version(self, binary=None, webdriver_binary=None):
         """Retrieve the release version of the installed browser."""
         version_string = call(binary, "--version").strip()
-        m = re.match(r"Mozilla Firefox (.*)", version_string)
+        m = re.match(br"Mozilla Firefox (.*)", version_string)
         if not m:
             return None
-        return m.group(1)
+        return m.group(1).decode()
```

Comment:
I'm not an expert on 2->3 migration, so this is a genuine question: what's the best practice in terms of dealing with stdio in general? Using a `br` regex seems weird *to me*. I *feel like* decoding the output before matching would be more intuitive.

Reply:
This is a really good point. Have to say that I'm not sure. Initially I was using six.ensure_str() before matching but then changed my mind after seeing "br" regex used in another place of the file... let me dig this a bit more.

Reply:
I did this mostly to avoid decoding the whole thing, which really is premature optimization (or dealing with the fact there might somehow be other non-ASCII data in the output). Let's just decode the whole thing.

Reply:
This is now addressed in https://github.com/web-platform-tests/wpt/pull/21656